cloud

```diff
@@ -1252,4 +1274,4 @@ function execute() {
                     --submitters="${submitters[*]@Q}"
 }
 
-execute "$@"
+execute "$@"
```

Comment:
Could you please make sure this commit doesn't remove the newline from the end of the file?

Maybe setup your editor to not remove the newline from the last line automatically, if it's doing that? Otherwise we will continue seeing this.